### PR TITLE
CM-998: Generate Images in C# Using: ImageMagick, SkiaSharp, and ImageSharp.

### PR DESCRIPTION
**Pull request by Emmanuel-Adom**

### CODE REVIEW

**Comment by Emmanuel-Adom** [comment]
Will fix them, Thank you @AndreCarvalho .

File: dotnet-client-libraries/ImageMagickSkiaSharpAndImageSharp/Tests/ImageMagickLibraryLiveTest.cs [hunk 1]
```diff
@@ -0,0 +1,126 @@
+using ImageMagick;
+using ImageMagickLibrary;
```
[the remark this answers is not in the document]
Comment (Emmanuel-Adom) [reply dotnet-client-libraries/ImageMagickSkiaSharpAndImageSharp/Tests/ImageMagickLibraryLiveTest.cs:2]: These namespaces are needed only in this file though.

File: dotnet-client-libraries/SystemDrawingCommonAndAForgeDotNet/Tests/SystemDrawingCommonLibraryLiveTest.cs [hunk 2]
```diff
@@ -0,0 +1,112 @@
+﻿using SystemDrawingCommonLibrary;
+
+namespace Tests;
+
+public class SystemDrawingCommonUnitTest
```
[the remark this answers is not in the document]
Comment (Emmanuel-Adom) [reply dotnet-client-libraries/SystemDrawingCommonAndAForgeDotNet/Tests/SystemDrawingCommonLibraryLiveTest.cs:5]: This file is for the previous project though. 
Marinko has thrashed the article and closed the pull request after all the hard work I've done.
You are also reviewing a file from it here.
Honestly, I'm not happy with what is going on at all.

**Comment by Emmanuel-Adom** [comment]
retest this please

**Comment by jshergal** [comment]
Closing this as we will be creating three separate articles for this Jira.

**Pull request closed without merging**